### PR TITLE
Add site for v1.0.0-alpha.2

### PR DESCRIPTION
**Pull request by nrb**
* Make the site for v1.0.0-alpha.2
* Update version warning header to include logic for alphas

Signed-off-by: Nolan Brubaker <brubakern@vmware.com>

### CODE REVIEW

**Comment by nrb** [comment]
I've also generated updated `master` docs. Want them in this PR, or separated?

**Comment by skriss** [comment]
no preference

File: master/README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ See [the list of releases][6] to find out about feature changes.
 
 
 [29]: https://heptio.github.io/velero/
-[30]: /troubleshooting.md
+[30]: /docs/troubleshooting.md
```
Comment (skriss) [inline master/README.md]: I think these links need to be tweaked manually to not have `/docs` in them, right?

File: master/about.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ This allows restore functionality to work in a cluster migration scenario, where
 Likewise, if a backup object exists in Kubernetes but not in object storage, it will be deleted from Kubernetes since the backup tarball no longer exists.
 
 [10]: hooks.md
-[19]: /img/backup-process.png
+[19]: /docs/img/backup-process.png
```
Comment (skriss) [inline master/about.md]: ditto re: removing /docs

**Pull request merged**